fiftyone/core/collections.py

```
@@ -10268,7 +10268,13 @@ def aggregate(self, aggregations):
 
         # Parse batch results
         if batch_aggs:
-            result = list(_results[0])
+            try:
+                result = _results[0].to_list()
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)